src/classes/BDI_DataImportBatch_SEL.cls

```diff
@@ -60,7 +60,8 @@ public with sharing class BDI_DataImportBatch_SEL {
                     Records_Failed__c,
                     Records_Successfully_Processed__c,
                     Post_Process_Implementing_Class__c,
-                    Run_Opportunity_Rollups_while_Processing__c
+                    Run_Opportunity_Rollups_while_Processing__c,
```

Comment:
@vmartinov-sforg just curious, does this resolve the issue we've been seeing on Batch clone and Batch edit when it's not through the Gift Entry interface?

Reply:
@ptewson-sfdo Can you remind me what issue you're referring to?